examples/lifecycleDemo.html

```diff
@@ -136,12 +137,13 @@ <h1>Lifecycle Events Log:</h1>
                     // Simulating bootstrap delay of 500ms
                     window.setTimeout(() => {
                         myClientApp.lifecycle.bootstrapped();
-
                         myClientApp.alerting.showToastPopup(
                             lifecycleStatusMessageTitle,
-                            'Bootstrap Complete (500ms delay)', {
-                                id: lifecycleStatusMessageId,
-                                type: 'success'
+                            `Bootstrap Complete (500ms delay):lifecycleStatusMessageId-bootstrap-${count}`, {
+                                id: `lifecycleStatusMessageId-bootstrap-${count++}`,
+                                type: 'success',
+                                showCloseButton: true,
+                                timeout: 0
```

Comment:
I wonder if we shouldn't add a querystring parameter to the url so that these "sticky" notifications aren't annoying to users using the examples in other contexts.  I personally use this example for manual regression testing.

Maybe add something like requireManualToastDismissal=true?